src/main/java/bhs/devilbotz/subsystems/DriveTrain.java

```diff
@@ -227,6 +254,23 @@ public void periodic() {
 
 
     }
+    // TODO: Remove
+    // @Log
+    // public double getleftVoltage(){ 
+    //     return leftMaster.getMotorOutputVoltage();
+    // }
+    // @Log
+    // public double getrightVoltage(){ 
+    //     return rightMaster.getMotorOutputVoltage();
+    // }
+    // @Log
+    // public double getleftVoltage1(){ 
+    //     return leftFollower.getMotorOutputVoltage();
+    // }
+    // @Log
+    // public double getrightVoltage1(){ 
+    //     return rightFollower.getMotorOutputVoltage();
+    // }
```

Comment:
Still marked for removal, if it is not needed, then please remove it, otherwise disregard this message.